packages/journey-manager/src/index.ts

```diff
@@ -126,17 +126,48 @@ export interface RunOutput {
   client: Client;
 }
 
+/**
+ * Journey run options
+ */
+export interface Options {
+  /**
+   * Digression detection callback
+   */
+  onDigression?: (client: Client) => void;
+}
+
 /**
  * Run the multimodal journey
  * @param config - The voice compass configuration
  * @param triggers - The triggers dictionary, downloaded from the Dialog Studio desktop app
+ * @param options - Journey run configuration
  * @returns an object containing a teardown function and the multimodal client.
  */
-export const run = (config: Config, triggers: Triggers): RunOutput => {
+export const run = (
+  config: Config,
+  triggers: Triggers,
+  options?: Options,
```

Comment:
Kind of weird having both `Config` and `Options`, they both sound kind of like bags of stuff. Maybe it would be nicer if we merged them?

Reply:
The `Config` is the same config that the regular voice compass SDK is using, we could do some kind of `Config & Options` thing but I think it's more awkward.

Reply:
Maybe then using a slightly better name like `ConnectionParameters` or some such